src/libraries/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -50,7 +51,7 @@ public void ReuseUnicastPort_CreateSocketSetOption()
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotQemuLinux))]
```

Comment:
Is this disable by design or due to a QEMU bug?

If this is disabled due to QEMU bug, it would be more appropriate to use `[ActiveIssue("https://....", typeof(PlatformDetection), nameof(PlatformDetection.IsQuemuLinux))]`.

Reply:
I think it's again Qemu issue but I couldn't prove exactly which one. It may has something to do with https://gitlab.com/qemu-project/qemu/-/issues/1837. There is chance it's new problem that actually I should report to Qemu (I have already reproducer in C but just postponed reporting). That's why test is conditionally disabled without any reference to existing issue.

Reply:
What Jan means is that we need a downstream tracking issue in dotnet/runtime (ideally referencing the quemu issue you shared) to remember that we can remove this workaround when the upstream is fixed.

With that issue being opened, we can use `ActiveIssue` instead of `ConditionalFact`, making the tracking issue visible in our test code.

Would you mind opening such an issue?

Reply:
Now I see. Sure, I can do that.